R/style_guides.R

```diff
@@ -75,7 +75,8 @@ tidyverse_style <- function(scope = "tokens",
         add_line_break_before_curly_closing,
       partial(
         set_line_break_after_opening_if_call_is_multi_line,
-        except_token = "COMMENT"
+        except_token_after = "COMMENT",
```

Comment:
Well you can do `style_text("a <- 3", scope = "tokens")`, so it is a top-level argument trough `...`